package-lock.json

```diff
@@ -5,8 +5,11 @@
     "packages": {
         "": {
             "devDependencies": {
+                "@popperjs/core": "^2.11.8",
                 "axios": "^1.1.2",
+                "bootstrap": "^5.3.2",
                 "laravel-vite-plugin": "^0.8.0",
+                "sass": "^1.69.0",
```

Comment:
this changes should be committed with package.json but the package.json has been committed before.
which is odd !!!